src/lib/app/types/layout.ts

```diff
@@ -15,3 +17,10 @@ export type ActionButton = {
     disabled?: boolean
     loading?: boolean
 }
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export type FilterCheckbox = {
+    label: string
+    onChange: (..._: any[]) => void
+    name: string 
```

Comment:
please call this `key` or `id`

Reply:
Name was removed, because its no longer needed. I simplified the solution a lot.